glue-codes/labview/src/WaveTank.f90

```diff
@@ -1,4 +1,31 @@
-
+!**********************************************************************************************************************************
+! LICENSING
+! Copyright (C) 2015-2016  National Renewable Energy Laboratory
+!
+!    This file is part of InflowWind.
```

Comment:
Update: "This file is the interface to a wave tank experiment set up at NREL." or something along these lines.

Reply:
Could be removed in favor of the description below.

Reply:
Whoops.  I didn't catch the `InflowWind` note during my copy/paste.